### DISCUSSION TITLE
keri-dev 02/01/2024

### DISCUSSION THREAD

**Discussion by pfeairheller** [WebOfTrust/keri#65] [Meetings]
# KERI Implementors Call 02/01/2024

- KERIpy
    - https://github.com/WebOfTrust/keripy/pull/661
    - https://github.com/WebOfTrust/keripy/pull/660
    - https://github.com/WebOfTrust/keripy/pull/640
    - https://github.com/WebOfTrust/keripy/pull/628
- KERIA
    - https://github.com/WebOfTrust/keria/pull/179
    - https://github.com/WebOfTrust/keria/pull/178
- Signify-TS
    - https://github.com/WebOfTrust/signify-ts/pull/203
    - https://github.com/WebOfTrust/signify-ts/pull/206
    - https://github.com/WebOfTrust/signify-ts/pull/209
    - https://github.com/WebOfTrust/signify-ts/pull/208
    - https://github.com/WebOfTrust/signify-ts/pull/207
    - https://github.com/WebOfTrust/signify-ts/pull/202
    - https://github.com/WebOfTrust/signify-ts/pull/189
    - https://github.com/WebOfTrust/signify-ts/pull/188
- Signifypy
    - https://github.com/WebOfTrust/signifypy/pull/105
    - https://github.com/WebOfTrust/signifypy/pull/104
- CESR-TS
    - None 
- CESRide
    - https://github.com/WebOfTrust/cesride/pull/167
    - https://github.com/WebOfTrust/cesride/pull/166
    - https://github.com/WebOfTrust/cesride/pull/165
    - https://github.com/WebOfTrust/cesride/pull/161
    - https://github.com/WebOfTrust/cesride/pull/155
    - https://github.com/WebOfTrust/cesride/pull/154
    - https://github.com/WebOfTrust/cesride/pull/149
    - https://github.com/WebOfTrust/cesride/pull/119
- did:webs
    - https://github.com/trustoverip/tswg-did-method-webs-specification/pull/132
    - https://github.com/trustoverip/tswg-did-method-webs-specification/pull/128
    - https://github.com/trustoverip/tswg-did-method-webs-specification/pull/124
    - https://github.com/trustoverip/tswg-did-method-webs-specification/pull/131
    - https://github.com/trustoverip/tswg-did-method-webs-specification/pull/130
    - https://github.com/trustoverip/tswg-did-method-webs-specification/pull/129
- did:webs Resolver
    - https://github.com/hyperledger-labs/did-webs-resolver/pull/44
    - https://github.com/hyperledger-labs/did-webs-resolver/pull/42
- KERIsse
- Browser Extension
    - White labeling https://github.com/WebOfTrust/signify-browser-extension/issues/17#issuecomment-1920166632
    - Localization https://github.com/WebOfTrust/signify-browser-extension/issues/82#issuecomment-1920169093

## Discussion

- Status of KERIA/Signify Release Candidates
    - Nothing to discuss for now

**Comment by clackwork** [comment]
I submit this PR for review, removes dozens of critical security flaws:
- https://github.com/WebOfTrust/vLEI/pull/83

**Reply by clackwork** [reply]
On related note is the possibility of establishing a Snyk dashboard for WebOfTrust/Keri team, if no such setup is already available:
- https://app.snyk.io/org/weboftrust/

This is offered gratis for libre software projects, and anyone may request access.
